networks/base/indexing.csv

```diff
@@ -0,0 +1,18 @@
+slug,provider,chain,address,technology,plan,trial,accessPrice,queryPrice,uptimeSla,bandwidthSla,blocksBehindSla,supportSla,starred,availableApis,limitations,securityImprovements,monitoringAndAnalytics,regions,additionalFeatures,verifiedUptime,verifiedLatency,verifiedBlocksBehindAvg,actionButtons
+thegraph-mainnet,The Graph,mainnet,null,Subgraph Indexing,Subscription-based,FALSE,0,GRT Fee,null,null,null,null,TRUE,"[""GraphQL""]","[""Rate limits apply""]","[""DDoS Protection"",""IP Protection""]","[""Subgraph logs"",""Usage analytics""]","[""Global""]","[""Decentralized Subgraphs"", ""Custom Queries""]",null,null,null,"[""[Buy](https://thegraph.com)"", ""[Docs](https://thegraph.com/networks/base/?subnetwork=base-mainnet)""]"

```

Comment:
Let's keep naming of the technology consistent everywhere, which would be The Graph instead of Subgraph Indexing